t/43-cli-archive.t

```diff
@@ -51,7 +51,62 @@ subtest 'Help' => sub {
     like $stdout, qr/Usage: openqa-cli archive/, 'help';
 };
 
+subtest 'Defaults' => sub {
+    my $archive = OpenQA::CLI::archive->new;
+    is $archive->apibase,   '/api/v1',          'apibase';
+    is $archive->apikey,    undef,              'no apikey';
+    is $archive->apisecret, undef,              'no apisecret';
+    is $archive->host,      'http://localhost', 'host';
+};
+
+subtest 'Host' => sub {
+    my $archive = OpenQA::CLI::archive->new;
+    eval { $archive->run('--host', 'openqa.example.com') };
+    like $@, qr/Usage: openqa-cli archive/, 'usage';
+    is $archive->host, 'https://openqa.example.com', 'host';
+
+    eval { $archive->run('--host', 'http://openqa.example.com') };
+    like $@, qr/Usage: openqa-cli archive/, 'usage';
+    is $archive->host, 'http://openqa.example.com', 'host';
+
+    eval { $archive->run('--osd') };
+    like $@, qr/Usage: openqa-cli archive/, 'usage';
```

Comment:
Maybe I'm just confused... but why are you checking for `Usage` if this is a valid option? 🤔

Reply:
The option is valid, but there's required parameters missing.

Reply:
Ah. How about iterating over a list of parameters that are valid but insufficient on their own? And drop the check for the value - it's irrelevant anyway at this point since you can't continue.

Reply:
I don't really see the point, it wouldn't test any extra logic.

Reply:
yeah, I also don't see the need, keep it simple :)

Reply:
I can't tell if you agree or disagree - I suggested to simplify the test code by iterating over the parameters without checking the values

Reply:
Oh, yes. I get it. Ok, I don't mind either way.

Reply:
I don't get it. How would iterating simplify anything?